// SPDX-License-Identifier: Apache-2.0
syntax = "proto3";

package gluten;

option java_package = "org.apache.gluten.proto";
option java_multiple_files = true;

message IcebergNestedField {
  int32 id = 1;
  repeated IcebergNestedField children = 2;
}